src/cargo/ops/cargo_compile/mod.rs

```diff
@@ -553,8 +553,12 @@ pub fn create_bcx<'a, 'gctx>(
                     .collect(),
             });
         }
+        let requested_units = root_units.iter().map(|unit| unit_to_index[unit]).collect();
         let elapsed = ws.gctx().creation_time().elapsed().as_secs_f64();
-        logger.log(LogMessage::UnitGraphFinished { elapsed });
+        logger.log(LogMessage::UnitGraphFinished {
+            requested_units,
```

Comment:
Why in the finished event? I feel like for json messages, we'd want this before the build started

Reply:
Same rename question <https://github.com/rust-lang/cargo/pull/16414#discussion_r2629626683>.

I am fine we moving forward with a temporary plan (like `build_target` and `platform_target`) if you have any.

Reply:
Ah. That makes sense. Thanks for the clarification